SUMMARY.md

```diff
@@ -70,10 +70,13 @@
   * [Migration to AEPPlaces v2](foundation-extensions/places/migration-v2.md)
   * [Places API Reference](foundation-extensions/places/places-usage-reference.md)
   * [ACPPlaces \(iOS\) Reference](foundation-extensions/places/adobe-places.md)
-* [Profile](foundation-extensions/profile/README.md)
-  * [Migrating to AEPUserProfile](foundation-extensions/profile/acpuserprofile-aepuserprofile.md)
+* [Adobe Experience Platform Profile](foundation-extensions/profile/README.md)
+  * [Migration to AEPUserProfile](foundation-extensions/profile/acpuserprofile-aepuserprofile.md)
   * [Profile API reference](foundation-extensions/profile/profile-api-references.md)
   * [Profiles event reference](foundation-extensions/profile/profiles-event-reference.md)
+* [Adobe Experience Platform Target](foundation-extensions/target/README.md)
```

Comment:
Do all Swift extensions go under `Mobile Foundations`? AJO is under `Experience Cloud`. How are we deciding the sections?

Reply:
Also, now Target API reference is in two sections. What's our plan here?

Reply:
What do you think of adding a link to the ACPTarget docs? Or is this too confusing?

Reply:
I would say this should go under Experience Cloud which captures the solution extensions.